pkg/types/vsphere/conversion/installconfig.go

```diff
@@ -0,0 +1,76 @@
+package conversion
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/vsphere"
+	"github.com/sirupsen/logrus"
+)
+
+func ConvertInstallConfig(config *types.InstallConfig) error {
```

Comment:
@rvanderp3 @bostrt wdyt?

Reply:
to me deprecated implies that we should still honor the legacy spec through 4.13 so I think this makes sense.